llmspec/llmspec.py

```diff
@@ -46,9 +46,6 @@ class LanguageModelInfo(msgspec.Struct):
 
     low_cpu_mem_usage: bool = True
 
-    # model structure
-    is_encoder_decoder: bool = True
```

Comment:
Then how to support different structures?

Reply:
Inspect the loaded model directly.